sharing/resource_share.go

```diff
@@ -2,9 +2,9 @@ package sharing
 
 import (
 	"context"
+	"sort"
 
 	"reflect"
-	"sort"
 
```

Comment:
unnecessary change

Reply:
Yeah, I removed sorting from all places but we need this for our unit test so added it again, this change happened due to that.